### ISSUE TITLE
Stuck at loading screen

### ISSUE THREAD

**Issue by hoangphat258** [yuliskov/SmartYouTubeTV#733]
I played a video and press the down-arrow button. The app stucks at loading and I can’t browse the video until I press back button to the searching screen

**Comment by Rin-Rem** [comment]
Yes, the loading screen that pops up is LITE UI. If you only PRO then for some reason the LITE seems to mix with PRO and I have no idea why. It starts happening randomly. And I can never figure out why or how to fix this issue.

**Comment by Liongkong** [comment]
Is your issue like, while watching the video and you press down to check for other video. Then when you press up it will turn into black screen with loading word but the sound still playing?

I can't turn back to the current watching video. Need to press all back and select the video that I'm watching to continue.

How to fix that issue?

**Comment by hoangphat258** [comment]
> Is your issue like, while watching the video and you press down to check for other video. Then when you press up it will turn into black screen with loading word but the sound still playing?
> 
> I can't turn back to the current watching video. Need to press all back and select the video that I'm watching to continue.
> 
> How to fix that issue?

yup, I got the same error and I don't know how to fix. I think this is a bug

**Comment by yuliskov** [comment]
Pro has been moved to the new app named SmartTube Next

Regards, Yuriy
